image-builder/builder-services/src/main/java/io/hyscale/builder/cleanup/services/impl/DeleteAll.java

```diff
@@ -34,9 +34,11 @@
 
 	@Override
 	public void clean(ServiceSpec serviceSpec) {
-		
-		CommandExecutor.execute(imageCommandProvider.getAllImageDeleteCommand(new HashSet<String>(
-				Arrays.asList(CommandExecutor.executeAndGetResults(imageCommandProvider.getAllImageCommand())
-						.getCommandOutput().split("\\s+")))));
+
+		String getAllimageCommand = imageCommandProvider.getAllImageCommand();
+		String[] imageIds = CommandExecutor.executeAndGetResults(getAllimageCommand).getCommandOutput().split("\\s+");
+		String allImageDeleteCommand = imageCommandProvider
+				.getAllImageDeleteCommand(new HashSet<String>(Arrays.asList(imageIds)));
```

Comment:
Better naming conventions for the methods , get all image commands sounds like we are deleting all images but we are deleting only hyscale labeled images.